pkg/auth/auth_context.go

```diff
@@ -172,3 +172,16 @@ func GetOauth2Config(options config.OAuthOptions) (oauth2.Config, error) {
 		},
 	}, nil
 }
+
+func GetL5Oauth2Config(mainConfig *oauth2.Config) oauth2.Config {
```

Comment:
this is admin public. why are we including l5 specific logic here?

Reply:
Can we port this to the Lyft private version?